integration/base_ver_2_app/package.json

```diff
@@ -39,7 +39,7 @@
     "hammerjs": "2.0.8",
     "intl": "1.2.5",
     "minimatch": "3.0.4",
-    "moment": "2.15.2",
+    "moment": "2.20.1",
```

Comment:
This one I think needs to carry the original value, as it's purpose is to check backwards compatibility with the released apps.